REFERENCE.md

```diff
@@ -141,3 +141,11 @@ Trust server certificate
 
 Default value: `false`
 
+##### `keep_local_changes`
```

Comment:
We can remove from line number 144 to 150. It will be generated automatically with stringify during the release step.